### PR TITLE
W-10945772: Fix broken alt-text syntax in images

### PR DESCRIPTION
**Pull request by fermujica**

### CODE REVIEW

File: modules/ROOT/pages/mq-tutorial.adoc [hunk 1]
```diff
@@ -422,18 +422,18 @@ The queue must be in an environment other than Design.
 image::mq-tutorial-url.png[Copy button in the Destinations page]
 . Copy the client app ID and client secret from the *Client Apps* page in Anypoint MQ to Studio:
 +
-.The screenshot shows (*1*) the *Client App ID* field, (*2*) *Show* button, and (*3*) *Copy* button.  
+.The screenshot shows (*1*) the *Client App ID* field, (*2*) *Show* button, and (*3*) *Copy* button.
 image::mq-tutorial-client-apps-detail.png[Client App ID field, Show and Copy buttons]
```
Comment (valkyrie69) [inline modules/ROOT/pages/mq-tutorial.adoc:426]: Does this one need quotes too since it has a comma?

Comment (valkyrie69) [inline modules/ROOT/pages/mq-tutorial.adoc:426]:
```suggestion
image::mq-tutorial-client-apps-detail.png["Client App ID field, Show and Copy buttons"]
```

File: modules/ROOT/pages/mq-tutorial.adoc [hunk 2]
```diff
@@ -422,18 +422,18 @@ The queue must be in an environment other than Design.
...
 +
 .. In the *MQ* page, click *Client Apps* and click `DemoClientApp` to display the details pane.
 .. In the *MQ* details pane, click the *Copy* button for the *Client App ID* field and paste the value in the *Client App ID* field in the *Anypoint MQ Config* window.
 .. In the *MQ* details pane, click the *Copy* button for the *Client Secret* field and paste the value in the *Client Secret* field in *Anypoint MQ Config* window:
 +
-.The screenshot shows (*1*) the *URL* field, (*2*) the *Client App ID* field, (*3*) the *Client Secret* field, and (*4*) the *Test Connection* button.  
+.The screenshot shows (*1*) the *URL* field, (*2*) the *Client App ID* field, (*3*) the *Client Secret* field, and (*4*) the *Test Connection* button.
 image::mq-tutorial-studio7-mq-config.png[URL, Client App ID, and Client Secret fields and Test Connection button in Anypoint MQ Config window]
```
Comment (valkyrie69) [inline modules/ROOT/pages/mq-tutorial.adoc:433]:
```suggestion
image::mq-tutorial-studio7-mq-config.png["URL, Client App ID, and Client Secret fields and Test Connection button in Anypoint MQ Config window"]
```

?

File: modules/ROOT/pages/mq-tutorial.adoc [hunk 3]
```diff
@@ -624,18 +624,18 @@ image::mq-tutorial-studio6-set-payload-props.png[Payload in the Value field]
 image::mq-tutorial-url.png[Copy button on the Destinations page]
 . Copy the client app ID and client secret from the *Client Apps* page in Anypoint MQ to Studio:
 +
-.The screenshot shows (*1*) the *Client App ID* field, (*2*) *Show* button, and (*3*) *Copy* button.  
+.The screenshot shows (*1*) the *Client App ID* field, (*2*) *Show* button, and (*3*) *Copy* button.
 image::mq-tutorial-client-apps-detail.png[Client App ID field, Show and Copy buttons]
```
Comment (valkyrie69) [inline modules/ROOT/pages/mq-tutorial.adoc:628]:
```suggestion
image::mq-tutorial-client-apps-detail.png["Client App ID field, Show and Copy buttons"]
```

?

File: modules/ROOT/pages/mq-tutorial.adoc [hunk 4]
```diff
@@ -624,18 +624,18 @@ image::mq-tutorial-studio6-set-payload-props.png[Payload in the Value field]
...
 +
 .. In the *MQ* page, click *Client Apps* and click `DemoClientApp` to display the details pane.
 .. In the *MQ* details pane, click the *Copy* button for the *Client App ID* field and paste the value in the *Client App ID* field in the *Anypoint MQ Configuration* window.
 .. In the *MQ* details pane, click the *Copy* button for the *Client Secret* field and paste the value in the *Client Secret* field in *Anypoint MQ Configuration* window:
 +
-.The screenshot shows (*1*) the *URL* field, (*2*) the *Client App ID* field, and (*3*) the *Client Secret* field.  
+.The screenshot shows (*1*) the *URL* field, (*2*) the *Client App ID* field, and (*3*) the *Client Secret* field.
 image::mq-tutorial-studio6-mq-config.png[URL, Client App ID, and Client Secret fields in Anypoint MQ Configuration window]
```
Comment (valkyrie69) [inline modules/ROOT/pages/mq-tutorial.adoc:635]:
```suggestion
image::mq-tutorial-studio6-mq-config.png["URL, Client App ID, and Client Secret fields in Anypoint MQ Configuration window"]
```

**Comment by fermujica** [comment]
Handled in https://github.com/mulesoft/docs-mq/pull/101

**Pull request closed without merging**